doc/Known_issues/CredSSP_error_bestiary.md

```diff
@@ -0,0 +1,28 @@
+# I get authentication error "CredSSP encryption Oracle remediation" when trying to RDP into a Windows machine from another Windows machine
+
+![CredSSP encryption Oracle remediation](CredSSPError.png)
+
+
+If you see the above error, please try the following two options:
+
+In this document the windows machine from where you are trying the RDP is called "client" and the
+windows 2016 machine you are trying an RDP into is called a "server".
+
+## Changing the options in the server itself:
+
+warning: Please note that this makes the server vulnerable.
+
+This is the safer option as it does not make the client vulnerable.
+On the Server, please start Server Manager.Go to Local Server and click on Remote Desktop.
+Make sure that the checkbox that says "allow connections only from computers running remote desktop with Network Level Authentication" is not checked.
+![System Properties]( SystemProperties.png)
+
+## Changing the options on the client:
+
+Warning: Please note that this makes the client vulnerable.
+On the client machine,
+1.run gpedit.msc
```

Comment:
Extra newlines need to be added between list points, otherwise they're all rendered in one line:

https://github.com/Juniper/contrail-windows-docs/blob/74a96feca350e2633967174013e21b36e4487a5d/doc/Known_issues/CredSSP_error_bestiary.md#changing-the-options-on-the-client

Reply:
FYI Github has markdown preview if you press the "View file" button in the top right of a file:

![image](https://user-images.githubusercontent.com/9220158/48895447-620d6800-ee45-11e8-93e8-60e78af1ad2e.png)

Reply:
ok